opcua/server/server.py

```diff
@@ -318,12 +318,12 @@ def get_namespace_index(self, uri):
         uries = self.get_namespace_array()
         return uries.index(uri)
 
-    def get_event_object(self, etype=ua.ObjectIds.BaseEventType, source=ua.ObjectIds.Server):
+    def get_event_object(self, event=ua.BaseEvent()):
```

Comment:
I agree, but this is not implementable without much more effort, since we have somehow come to attributes of the event.... In this way is also dynamic... just quicker solution... I could try other way around but it will take some time...